guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -3,31 +3,51 @@
 
 Each provisioning type requires some network configuration. Use this chapter to configure network services in {ProjectServer}'s integrated {SmartProxy}.
 
-New hosts must have access to your {SmartProxyServer}. A {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
+New hosts must have access to your {SmartProxyServer}. {SmartProxyServer} can be either {ProjectServer}'s integrated {SmartProxy} or an external {SmartProxyServer}. You might want to provision hosts from an external {SmartProxyServer} when the hosts are on isolated networks and cannot connect to the {ProjectServer} directly, or when the content is synchronized with the {SmartProxyServer}. Provisioning using the external {SmartProxyServer} can save on network bandwidth.
```

Comment:
To avoid using  {ProjectServer}'s in the possessive form I suggest removing it and only keep the following:
New hosts must have access to your integrated {SmartProxy} or external {SmartProxyServer}.